crates/solvers/config/example.paraswap.toml

```
@@ -1,6 +1,8 @@
+node-url = "http://localhost:8545"
 absolute-slippage = "40000000000000000" # Denominated in wei, optional
 relative-slippage = "0.1" # Percentage in the [0, 1] range
```

To be consistent with the other changes.
```suggestion
relative-slippage = "0.001" # Percentage in the [0, 1] range
```